components/table/stories/template.js

```diff
@@ -204,65 +285,75 @@ export const Template = ({
 		id=${ifDefined(id)}
 		role=${ifDefined(useCheckboxCell ? "grid" : useDivs ? "table" : undefined)}
 		aria-multiselectable=${ifDefined(useCheckboxCell ? "true" : undefined)}
-		style="max-width: 800px;"
 	>
 		<${theadTag}
 			class="${rootClass}-head"
 			role=${ifDefined(useDivs ? "rowgroup" : undefined)}
 		>
 			<${rowTag}
 				role=${ifDefined(useDivs ? "row" : undefined)}
+				class="${rootClass}-headRow"
 			>
 				${when(useCheckboxCell, () => html`
 					<${thTag}
 						class="spectrum-Table-headCell spectrum-Table-checkboxCell"
 						role=${ifDefined(useDivs ? "columnheader" : undefined)}
 					>
-						${Checkbox({
-							size,
-							isEmphasized: isEmphasized,
-							isChecked: false,
-							isIndeterminate: true,
-							customClasses: [`${rootClass}-checkbox`],
+						${when(selectionMode === "multiple", () => html`
+							${Checkbox({
+								size: "m",
+								isEmphasized: isEmphasized,
+								isChecked: false,
+								isIndeterminate: true,
+								customClasses: [`${rootClass}-checkbox`],
 						}, context)}
-					</${thTag}>`
+						`)}
+					</${thTag}>`,
 				)}
+				<${thTag}
+					class="${rootClass}-headCell is-sortable"
+					role=${ifDefined(useDivs ? "columnheader" : undefined)}
+					aria-sort="none"
+					tabindex="0"
+				>
+					${Icon({
+						iconName: "Sort",
```

Comment:
I added each of the new sort icons to each header cell.

Reply:
I wonder if we should remove these as far as the default display, and only have them within a Story related to sorting? Then we can also include any relevant docs for using sorting.

Reply:
Sure I can do that!